app/grandchallenge/archives/tasks.py

```diff
@@ -81,3 +83,90 @@ def update_archive_item_values(
         archive_item = ArchiveItem.objects.get(pk=archive_item_pk)
         archive_item.values.remove(*civ_pks_to_remove)
         archive_item.values.add(*civ_pks_to_add)
+
+
+def update_archive_item_update_kwargs(
```

Comment:
Could you add a docstring to this method? It's not really clear to me what this should do from the name